### PR TITLE
[MON-2]Implemented delete function

### PR DESCRIPTION
**Pull request by aa-kashk**

### CODE REVIEW

File: monitor/src/main/java/edu/asu/diging/monitor/core/service/impl/NotificationManager.java [hunk 1]
```diff
@@ -53,17 +55,23 @@ public boolean addRecipient(String name, String email) throws EmailAlreadyRegist
 		}
 		return true;
 	}
-	
+
+	@Override
+	public void deleteRecipient(String email) {
+		dbConnection.delete(email);
```
Comment (jdamerow) [inline monitor/src/main/java/edu/asu/diging/monitor/core/service/impl/NotificationManager.java:61]: let's add a test

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Jenkins, test this please.

**Comment by jdamerow** [comment]
Also, the delete icon should be right justified and the first line is a little thicker for the first two columns.
image

File: monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
-<a href="<c:url value="/admin/recipients/add"/>" class="btn btn-primary">Add Recipient</a>
+<a href="<c:url value="/admin/recipients/add"/>" class="btn btn-primary">Add
+	Recipient</a>
 <table class="table">
 	<thead>
-		<th scope="col">Name</th>
-		<th scope="col">Email</th>
+		<th scope="col" style="width:20%">Name</th>
+		<th scope="col" style="width:20%">Email</th>
+		<th style="width:100%"></th>
```
Comment (jdamerow) [inline monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp]: that would add up to 140%, which I"m pretty sure is invalid

File: monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp [hunk 3]
```diff
@@ -1,19 +1,35 @@
...
+							var="deleteUrl" />
+						<form action="${deleteUrl}" method="POST">
+							<input type="hidden" name="${_csrf.parameterName}"
+								value="${_csrf.token}" />
+							<button title="Delete Recipient" type="submit" class="btn-link">
+								<i class="fa fa-trash" aria-hidden="true"></i>
+							</button>
+						</form>
+					</div>
+				</sec:authorize></td>
```
Comment (jdamerow) [inline monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp]: indentation

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp (hunk 3 above)
Comment (jdamerow) [reply monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp]: and to reiterate my statement from before icon "should be right justified"

Comment (jdamerow) [reply monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp]: ?

Comment (aa-kashk) [reply monitor/src/main/webapp/WEB-INF/views/admin/showRecipient.jsp]: sorry, I used the command+shift+F option to format it. I hope it Is okay now

**Comment by jdamerow** [comment]
For the future, please make sure to name your branches according to our guidelines (prefix should be according to ticket type, e.g. in this case `story/MON-2` not `task/MON-2`).

**Pull request merged**